Signal/src/AppDelegate.m

```diff
@@ -410,7 +410,8 @@ - (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull N
 
         OutboundCallInitiator *outboundCallInitiator = [Environment getCurrent].outboundCallInitiator;
         OWSAssert(outboundCallInitiator);
-        return [outboundCallInitiator initiateCallWithHandle:phoneNumber];
+        [outboundCallInitiator initiateCallWithHandle:phoneNumber];
+        return YES;
```

Comment:
You're right - the invite flow would be best here.  I'm going to restore the return value to make that eventual work easier.